reference/media-types.md

```diff
@@ -387,6 +387,42 @@ responds to `foobar`.
 Praxis media types can declare attributes that are ordered collections of other
 media types or other Attributor types.
 
+### Praxis::Collection
+
+Praxis includes a helper, `Praxis::Collection.of(media_type)`, for working with  collections of a given media type. (*Note*: Previously, `Praxis::Collection.of` supported types that did not derive from `MediaType`. This usage has been deprecated and will be removed in a future version. Please use `Attributor::Collection.of` directly instead.)
+
+This defines a `<media_type>::Collection` type that is an `Attributor::Collection.of(<media_type>)`, that also includes  the `MediaTypeCommon` module. By default, Praxis will take the identifier of `<media_type>` and append a "collection=true" suffix to it.
```

Comment:
I'd start with:

`Praxis::Collection.of(media_type)` defines an inner class inside the `media_type` (i.e., `<media_type>::Collection`) that is an ...